### ISSUE TITLE
Documentation says second argument is optional, but TS compiler says its required

### ISSUE THREAD

**Issue by joshtynjala** [rgrove/parse-xml#23]
Docs say I can do this:

```ts
parseXml('<kittens fuzzy="yes">I like fuzzy kittens.</kittens>');
```

But I get an error:

> Expected 2 arguments, but got 1.

Workaround:

```ts
parseXml('<kittens fuzzy="yes">I like fuzzy kittens.</kittens>', undefined);
```

I think that the type definition needs to make the second parameter optional with `?`:

```ts
options?: ParserOptions
```

**Comment by rgrove** [comment]
You're right! Good catch.

**Comment by rgrove** [comment]
Published [4.0.1](https://github.com/rgrove/parse-xml/releases/tag/v4.0.1) with a fix.

**Comment by joshtynjala** [comment]
Thanks!
